src/modules/dashboard/components/ExpendituresDialog/ExpendituresDialog.tsx

```diff
@@ -73,26 +73,23 @@ const ExpendituresDialog = ({
   prevStep,
   colony: { colonyAddress },
   colony,
+  isVotingExtensionEnabled,
   nextStep,
 }: Props) => {
   const { walletAddress, username, ethereal } = useLoggedInUser();
+  const { isOneTxPaymentExtensionEnabled } = useEnabledExtensions({
+    colonyAddress,
+  });
 
   const allUserRoles = useTransformer(getAllUserRoles, [colony, walletAddress]);
 
   const hasRegisteredProfile = !!username && !ethereal;
   const canCreatePayment =
     hasRegisteredProfile &&
-    canAdminister(allUserRoles) &&
-    canFund(allUserRoles);
-
-  const { data: colonyExtensionsData } = useColonyExtensionsQuery({
-    variables: { address: colonyAddress },
-  });
+    ((canAdminister(allUserRoles) && canFund(allUserRoles)) ||
+      isVotingExtensionEnabled);
 
-  const canMakePayment =
-    colonyExtensionsData?.processedColony?.installedExtensions?.find(
-      ({ extensionId }) => extensionId === Extension.OneTxPayment,
-    ) || false;
+  const canMakePayment = isOneTxPaymentExtensionEnabled || false;
```

Comment:
Also, this variable naming is very bad, since we already have `canCreatePayment`, and this being called `canMakePayment` is very confusing....

Reply:
agree, update that too